services/provider/server/server.go

```diff
@@ -423,6 +420,48 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	cbpList := &rookCephv1.CephBlockPoolList{}
+	err = s.client.List(ctx, cbpList, client.InNamespace(s.namespace))
+	if err != nil {
+		return nil, fmt.Errorf("failed to list cephBlockPools in namespace. %v", err)
+	}
+	blockPoolMapping := []csiopv1a1.BlockPoolIdPair{}
+	for i := range cbpList.Items {
+		cephBlockPool := cbpList.Items[i]
+		if cephBlockPool.GetAnnotations()[util.BlockPoolMirroringTargetIDAnnotation] != "" {
+			blockPoolMapping = append(
+				blockPoolMapping,
+				[]string{
```

Comment:
Please use the proper type here:
```suggestion
				csiopv1a1.BlockPoolIdPair{
```